sal_interfaces/Forecast/Forecast_Telemetry.xml

```diff
@@ -0,0 +1,674 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+  <SALTelemetry>
+    <Subsystem>Forecast</Subsystem>
+    <EFDB_Topic>Forecast_metadata</EFDB_Topic>
+    <item>
+      <EFDB_Name>latitude</EFDB_Name>
+      <Description>The latitude of the forecast data</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>longitude</EFDB_Name>
+      <Description>The longitude of the forecast data</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>height</EFDB_Name>
+      <Description>The height of the forecast.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>meter</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timezone_abbrevation</EFDB_Name>
+      <Description>The abbrevation of the timezone.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>utc_timeoffset</EFDB_Name>
+      <Description>Hour offset from UTC</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>hour</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>modelrun_utc</EFDB_Name>
+      <Description>The time that the forecast was run</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>modelrun_updatetime_utc</EFDB_Name>
+      <Description>The time that the model was last updated</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>Forecast</Subsystem>
+    <EFDB_Topic>Forecast_trend_1h</EFDB_Topic>
+    <item>
+      <IDL_Type>double</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast time</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>forecast_time</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>360</Count>
+      <Description>Forecast temperature</Description>
```

Comment:
It is clear that this is the `Forecast` temperature so you can drop that prefix here and everywhere else.